auslib/admin/views/scheduled_changes.py

```diff
@@ -94,13 +94,18 @@ def _post(self, sc_id, form, transaction, changed_by):
             if (request.json and k in request.json):
                 what[k] = v.data
 
+        where = {"sc_id": sc_id}
         try:
-            self.sc_table.update({"sc_id": sc_id}, what, changed_by, form.sc_data_version.data, transaction)
+            self.sc_table.update(where, what, changed_by, form.sc_data_version.data, transaction)
+            signOffs = self.sc_table.signoffs.select(where=where, transaction=transaction, columns=["sc_id", "username"])
+            for signOff in signOffs:
+                self.sc_table.signoffs.delete(where={"sc_id": sc_id, "username": signOff["username"]},
+                                              changed_by=changed_by, transaction=transaction)
```

Comment:
This logic looks fine, but it belongs in ScheduledChangesTable.update. If we put it here, and other code comes along that calls into ScheduledChangesTable.update, we'd have to duplicate this logic (or forget, and introduce a bug).